setup/setupEnvironment.sh

```diff
@@ -178,4 +178,37 @@ GATEWAY_ID=$(cat /tmp/apigateway-import-api | grep id | awk '{print $2}' | xargs
 echo "Deploying the gateway with id of " ${GATEWAY_ID} " to prd"
 aws apigateway create-deployment --rest-api-id ${GATEWAY_ID} --stage-name prod
 
+## Create a script that will remove (most) all of the AWS resources created
```

Comment:
This is actually a really good idea...not sure why you closed this PR (except for the .template changes), but I'm implementing this functionality right now...it's really useful (although I'm breaking up this code into the segments -- each 'deletion logic' is created after the actual resource is created, rather than at the end. Thanks!